charmcraft/commands/build.py

```diff
@@ -312,20 +312,7 @@ def run(
 
         :returns: List of charm files created.
```

Comment:
This is not longer true, right?

Reply:
Indeed, I rewrote the entire doc string, thanks!